code-challenges.test.js

```diff
@@ -32,28 +41,106 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+
+describe("divisibleBy", () => {
+    it ("takes in a number and checks if it is divisible by three", () => {
+        expect(divisibleBy(object1)).toEqual((divisibleBy)("0 is divisible by three"));
+        expect(divisibleBy(object2)).toEqual((divisibleBy)("-7 is not divisible by three"))
+        expect(divisibleBy(object3)).toEqual((divisibleBy)("15 is divisible by three"))
+    })
+})
+
+
+// // b) Create the function that makes the test pass.
+
+// // process: declare a function named divisible by that takes in a number, and provides a conditional in which if the number is divisible by 3 it returns "this number is divisible by 3", and " this number is not divisible by three" if not
+
+
+const divisibleBy = (num) => {
+   if (num % 3 === 0) {
+    return "0 is divided by three" }
+    else if (num % 3 == 1) {
+        return "15 is divided by three"}
+     else if (num % 3 !==1) 
+        {return "-7 is not divisible by three"
+
+    }
+   }
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+describe("capitalArray", () => {
+    it ("takes in an array and capitalizes the first index of each string", () => {
+        expect(capitalArray(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(capitalArray(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
+// })
+
+// // // // error: ReferenceError: capitalArray is not defined
+
+// // // // b) Create the function that makes the test pass.
+// process: declare a function called capital array and return an array with the method .map along with the dot notation methods charAt with the argument at index zero, and then using .slice to combine the values using .at the index one, and .toLowerCase
+
+// const capitalArray = (array) => {
+//     return array.map((value) => value.charAt(0).toUpperCase() + value.slice(1))
```

Comment:
That is a weird, I copied this into my own VS code and it ran but great job finding another way to do this!